FireRender.Maya.Src/FireRenderGlobals.cpp

```diff
@@ -437,10 +452,32 @@ MStatus FireRenderGlobals::initialize()
 	Attribute::tahoeVersion = eAttr.create("tahoeVersion", "tahv", TahoePluginVersion::RPR1, &status);
 	eAttr.addField("RPR 1", TahoePluginVersion::RPR1);
 	eAttr.addField("RPR 2 (Experimental)", TahoePluginVersion::RPR2);
-
 	MAKE_INPUT_CONST(eAttr);
 	CHECK_MSTATUS(addAttribute(Attribute::tahoeVersion));
 
+	int textureCacheExists;
+	MGlobal::executeCommand("optionVar -ex RPR_textureCachePath", textureCacheExists);
+	if (textureCacheExists == 0)
+	{
+		MString workspace;
+		status = MGlobal::executeCommand(MString("workspace -q -dir;"), workspace);
+		workspace += "texture_cache";
+		MGlobal::executeCommand(MString("optionVar -sv RPR_textureCachePath") + workspace);
+
+		MObject defaultTextureCachePath = sData.create(workspace);
```

Comment:
Please use just attibute in order to work with optionVar RPR_textureCacheVariable after its creation. We may discuss